frontend/packages/component-library/src/video/FacadeBackground.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './video.module.scss';
+
+export interface FacadeProps {
+  /** Facade poster thumbnail */
+  posterThumbnail?: string;
+  onClick?: () => void;
```

Comment:
Nit: needs comment here
```suggestion
  /** Facade onClick */
  onClick?: () => void;
```